### PR TITLE
check out git submodules

### PR DESCRIPTION
**Pull request by simonschmeisser**
See here for documentation

https://github.com/dirk-thomas/vcstool/blob/ea4c777989fbdc71d0163beb0ec6ea69f524a70d/vcstool/commands/import_.py#L54

fixes issues when building eigenpy and likely others

### CODE REVIEW

**Comment by simonschmeisser** [comment]
I added two more optional commits that I have not verified to work but which might be interesting as well

File: prepare.sh [hunk 1]
```diff
@@ -37,4 +37,4 @@ cat ~/.sbuildrc
 echo "Checkout workspace"
 
 mkdir src
-vcs import --recursive --input  "$REPOS_FILE" src
+vcs import --recursive --shallow --input  "$REPOS_FILE" src
```
Comment (jspricke) [inline prepare.sh:40]: I think this would break the sha1 in the version number.

**Comment by simonschmeisser** [comment]
fixes https://github.com/v4hn/ros-o-builder/issues/15

File: prepare.sh (hunk 1 above)
Comment (v4hn) [reply prepare.sh:40]: Not the sha1, but the [tag-based version scheme](https://github.com/jspricke/ros-deb-builder-action/blob/52ed9a9257ca7597ce5e7bf87968b3f14e3e5901/build.sh#L71-L73), right?

**Comment by v4hn** [comment]
Technically, this does not fix the ros-o-builder issue because I use my fork there for now. :-)
I'll cherry-pick and adapt the first two commits though as they make sense to me.

File: prepare.sh [hunk 2]
```diff
@@ -37,11 +37,4 @@ cat ~/.sbuildrc
...
 mkdir src
-case $REPOS_FILE in
-  http*)
-    curl -sSL "$REPOS_FILE" | vcs import --recursive src
-    ;;
-  *)
-    vcs import --recursive src < "$REPOS_FILE"
-    ;;
-esac
+vcs import --recursive --input  "$REPOS_FILE" src
```
Comment (jspricke) [inline prepare.sh]: can you `git revert b228fee` first, then add this on top and then your other changes?

Comment (v4hn) [reply prepare.sh]: Makes sense. :+1:

Comment (v4hn) [reply prepare.sh]: Sorry, I take that back. [curl is still a dependency that should be installed](https://github.com/jspricke/ros-deb-builder-action/blob/dd94ff4e014a3a33d03db6cf13178ebb55ada964/build.sh#L25)

File: prepare.sh (hunk 1 above)
Comment (simonschmeisser) [reply prepare.sh]: Might be worthwhile to test at some point but dropped for now

**Pull request merged**